app/src/services/ShopService.ts

```diff
@@ -440,7 +471,25 @@ export const ShopService: ShopControllerSocket & DashboardControllerSocket = {
       console.error('Invalid date, earlier than today')
       throw new InvalidParamsError()
     }
-    return ReservationRepository.insertReservation(reservationDate, clientId, shopId, menuId, stylistId)
+
+    const gtedate = filterAndGetWantedDate(reservationDate, true, false)
+    const ltdate = filterAndGetWantedDate(reservationDate, false, true)
```

Comment:
passing boolean as an argument is a bad idea, it's easier to just return both values as an object from one function